src/core/server/pulse/client_wrappers/elasticsearch.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import uuid from 'uuid';
+import { IPulseElasticsearchClient } from './types';
+import { IClusterClient } from '../../elasticsearch';
+
+export class PulseElasticsearchClient implements IPulseElasticsearchClient {
```

Comment:
If we decide not to use the wrapper for Elasticsearch. Just remove the whole file!

Reply:
Let's keep the wrappers for now.